src/stdune/path.ml

```diff
@@ -146,7 +127,7 @@ end = struct
       Code_error.raise "Path.External.parent_exn called on a root path" []
     | Some p -> p
 
-  let mkdir_p ?perms p = Fpath.mkdir_p ?perms (to_string p)
+  let mkdir_p ?perms p = ignore (Fpath.mkdir_p ?perms (to_string p))
```

Comment:
It's good practice to annotate ignores as follow: `ignore (<expr> : <type>)`. Otherwise, if `mkdir_p` gets a new argument this code won't break.